### ISSUE TITLE
Document how to use `deployer generate dedicated-cluster` in  the creation proces of new clusters

### ISSUE THREAD

**Issue by GeorgianaElena** [2i2c-org/infrastructure#2187]
### Context

The template configuration files in `config/cluster/templates`, the:
```
├── aws
├── common
│   ├── support.secret.values.yaml
│   └── support.values.yaml
└── gcp
    └── cluster.yaml
```
 Can  now be used through the `deployer` to auto-generate specific files for new clusters on AWS and for new clusters and an initial hub on GCP.

### Proposal

- [x] Document how these templates, together with the `deployer generate dedicated-cluster` command can be used to make the process of adding a new cluster easier.
- [ ] Update or remove existing documentation like https://infrastructure.2i2c.org/hub-deployment-guide/deploy-support/configure-support/#create-a-support-values-yaml-file-in-your-chosen-cluster-folder as this is now replaced by the command completely.

Ref: https://github.com/2i2c-org/infrastructure/issues/1917

**Comment by GeorgianaElena** [comment]
Useful feedback from @pnasrat  about this one:

- 1. **We need to document it**
- 2. Originally posted by @pnasrat in https://2i2c.slack.com/archives/C01GLCC1VCN/p1677843574698329?thread_ts=1677772541.271159&cid=C01GLCC1VCN
_"I think in general having templated config and tooling makes sense to reduce operational work, I had to comment out the terraform generation as I had already done that
Features that would have been useful - skipping a step (eg provisioning for terraform/eksctl), a flag allowing overwriting a cluster.yaml,
This means integration with new cloud providers might ideally look like doing once manually with a test providers, write up manual steps, and writing deployer code necessary to do - a  test create/destroy with deployer and create the actual community cluster"_

**Comment by GeorgianaElena** [comment]
I updated the top comment to only cover the documentation part of the generator command as the command has been already extended to account for more providers (GCP and AWS currently). Also, I opened https://github.com/2i2c-org/infrastructure/issues/3294 as a follow-up to the comment above

**Comment by damianavila** [comment]
Ask @sgibson91 if she can pair with @GeorgianaElena on this one.

**Comment by GeorgianaElena** [comment]
> Document how these templates, together with the deployer generate dedicated-cluster command can be used to make the process of adding a new cluster easier.

This was actually solved by #3878 that standardized all three providers in the same document -> https://infrastructure.2i2c.org/hub-deployment-guide/new-cluster/new-cluster/

> Update or remove existing documentation like https://infrastructure.2i2c.org/hub-deployment-guide/deploy-support/configure-support/#create-a-support-values-yaml-file-in-your-chosen-cluster-folder as this is now replaced by the command completely.

This is the bit missing ⬆️  from considering this task done.
